integration-tests/src/test/java/oracle/kubernetes/operator/utils/Domain.java

```diff
@@ -1708,23 +1721,31 @@ private void changeClusterTypeInCreateDomainJobTemplate() throws Exception {
 
     // change CLUSTER_TYPE to CONFIGURED in create-domain-job-template.yaml for configured cluster
     // as samples only support DYNAMIC cluster
-    if (clusterType.equalsIgnoreCase("CONFIGURED")) {
-
-      // domain in image
-      if (domainMap.containsKey("domainHomeImageBase")
-          && domainHomeImageBuildPath.contains("wdt")) {
-        TestUtils.copyFile(
-            BaseTest.getProjectRoot()
-                + "/integration-tests/src/test/resources/wdt/config.cluster.topology.yaml",
-            BaseTest.getResultDir()
-                + "/docker-images/OracleWebLogic/samples/12213-domain-home-in-image-wdt/simple-topology.yaml");
-      } else {
-        // domain on pv
-        StringBuffer createDomainJobTemplateFile = new StringBuffer(BaseTest.getResultDir());
-        createDomainJobTemplateFile.append(
-            "/samples/scripts/create-weblogic-domain/domain-home-on-pv/create-domain-job-template.yaml");
-        TestUtils.exec("sed -i -e 's?DYNAMIC?CONFIGURED?g' " + createDomainJobTemplateFile);
+
+    // domain in image
+    if (domainMap.containsKey("domainHomeImageBase") && domainHomeImageBuildPath.contains("wdt")) {
+      String wdtTemplatePath =
+          BaseTest.getProjectRoot()
+              + "/integration-tests/src/test/resources/wdt/config.cluster.topology.yaml";
+      if (domainMap.containsKey("customWdtTemplate")) {
+        wdtTemplatePath = (String) domainMap.get("customWdtTemplate");
       }
+      TestUtils.copyFile(
+          wdtTemplatePath,
+          BaseTest.getResultDir()
+              + "/docker-images/OracleWebLogic/samples/12213-domain-home-in-image-wdt/simple-topology.yaml");
+      ExecResult exec =
+          TestUtils.exec(
+              "cat "
+                  + BaseTest.getResultDir()
+                  + "/docker-images/OracleWebLogic/samples/12213-domain-home-in-image-wdt/simple-topology.yaml");
+      logger.info(exec.stdout());
```

Comment:
change to debug or remove this after testing, it will be very verbose..

Reply:
done